pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -60,6 +64,9 @@ type nodeTopoInformer struct {
 	metricCache    metriccache.MetricCache
 	callbackRunner *callbackRunner
 
+	NodeResourceTopologyInformer cache.SharedIndexInformer
```

Comment:
how about `nodeResourceTopologyInformer` and `nodeResourceTopologyLister`, keep the naming consistent with the `nodeInformer` and `podsInformer` below.